### PR TITLE
PubWise Bid Adapter: clean up and updates to adUnit parsing

### PR DESCRIPTION
**Pull request by GLStephen**
Update for PubWise Bid Adapter.

### CODE REVIEW

File: modules/pubwiseBidAdapter.js [hunk 1]
```diff
@@ -532,9 +532,8 @@ function _cleanSlotName(slotName) {
 
 function _initConf(refererInfo) {
   return {
-    // TODO: do the fallbacks make sense here?
-    pageURL: refererInfo?.page || window.location.href,
-    refURL: refererInfo?.ref || window.document.referrer
+    pageURL: (refererInfo && refererInfo.referer) ? refererInfo.referer : window.location.href,
+    refURL: window.document.referrer
```
Comment (patmmccann) [inline modules/pubwiseBidAdapter.js:536]: Why aren't you using refererInfo. Ref here?

Comment (GLStephen) [reply modules/pubwiseBidAdapter.js:536]: @patmmccann not sure, I'll check, but looks like we should be

**Comment by GLStephen** [comment]
> one more question, could you solve for #8573 here? We're attempting to enforce all bidders pass:
> 
> auctionId as source.tid bidderRequest[i].transationId as imp[i].ext.tid

Sure thing

File: modules/pubwiseBidAdapter.js (hunk 1 above)
Comment (patmmccann) [reply modules/pubwiseBidAdapter.js:536]: refererInfo.ref and refererInfo.page have a bunch of logic attempting to determine them as of Prebid 7 merge

Comment (GLStephen) [reply modules/pubwiseBidAdapter.js:536]: This has been adjusted in the next update.

**Comment by GLStephen** [comment]
@patmmccann updates have been made

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**